src/mono/wasm/Wasm.Build.Tests/BuildPublishTests.cs

```diff
@@ -15,17 +15,23 @@ namespace Wasm.Build.Tests
 {
     public class BuildPublishTests : NativeRebuildTestsBase
     {
+        private readonly char unicodeChar = '煉';
+
         public BuildPublishTests(ITestOutputHelper output, SharedBuildPerTestClassFixture buildContext)
             : base(output, buildContext)
         {
         }
 
         [Theory]
-        [BuildAndRun(host: RunHost.Chrome, aot: false, config: "Release")]
-        [BuildAndRun(host: RunHost.Chrome, aot: false, config: "Debug")]
-        public void BuildThenPublishNoAOT(BuildArgs buildArgs, RunHost host, string id)
+        [BuildAndRun(host: RunHost.Chrome, aot: false, config: "Release", parameters: new object[] { false })]
+        [BuildAndRun(host: RunHost.Chrome, aot: false, config: "Debug", parameters: new object[] { false })]
+        [BuildAndRun(host: RunHost.Chrome, aot: false, config: "Release", parameters: new object[] { true })]
+        [BuildAndRun(host: RunHost.Chrome, aot: false, config: "Debug", parameters: new object[] { true })]
+        public void BuildThenPublishNoAOT(BuildArgs buildArgs, bool testsUnicode, RunHost host, string id)
         {
-            string projectName = $"build_publish_{buildArgs.Config}";
+            string projectName = testsUnicode ?
+                $"build_publish_{buildArgs.Config}{unicodeChar}" :
```

Comment:
We could even always use the unicodeChar in the the project name, instead of having separate tests.

Reply:
We could but then we don't know at the first glance if we messed up the Unicode chars or the publish/build when it starts to fail. That was my main motivation of making a switch there.

Reply:
Good thinking 👍